CONTRIBUTING.rst

```diff
@@ -83,23 +83,28 @@ Releases are created using `manual GitHub workflows
 <https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow>`_.
 As a maintainer, follow these steps:
 
-1. Create a new branch
-2. Bump the version defined in ``skops/__init__.py``
-3. Git grep for any TODO's that need fixing before the release (e.g.
+1. Check and update the ``docs/changes.rst``
+2. For a major release, create a new branch with the name "0.version.X", e.g.
+   "0.2.X". This branch will have all tags for all releases under 0.2.
+3. Bump the version defined in ``skops/__init__.py``
+4. Git grep for any TODO's that need fixing before the release (e.g.
```

Comment:
Can you explain this a little more? Maybe type an example command? 
Please ignore if it doesn't make any sense but if I were a new maintainer I might've been confused.

Reply:
added this.